chrome_gpu_bug_count.sh

```diff
@@ -7,7 +7,7 @@ BASEDIR=$(dirname "$0")
 INPUT="$BASEDIR/chrome_gpu_triage.json"
 SCRIPT="$BASEDIR/issue_counter.py"
 mkdir -p "$OUTDIR"
-OUTFILE="$OUTDIR/$(date --iso-8601).json"
+OUTFILE="$OUTDIR/$(date "+%Y-%m-%d-%H:%M").json"
```

Comment:
I'm happy to submit a CL to change it to that, if it'd make your logs/CSV upload process smoother!